### PR TITLE
[NativeIFC] Python 3.11 Fix

### PR DESCRIPTION
**Pull request by Syres916**
Not sure but I really think this is what you actually intended?

### CODE REVIEW

File: ifc_objects.py [hunk 1]
```diff
@@ -117,15 +117,15 @@ def __getstate__(self):
...
+        self.loads(state)
 
     def dumps(self):
-        return
+        return getattr(self, "Type", None)
 
     def loads(self, _state):
-        return
+        if state and hasattr(state, "Type"):
+            self.Type = state
```
Comment (yorikvanhavre) [inline ifc_objects.py:127]: this should be self.Type = _state, right?

```suggestion
            self.Type = _state
```

**Review by yorikvanhavre — CHANGES_REQUESTED** [review]
Yes that's perfect! Thanks!

**Comment by Syres916** [comment]
Both `__setstate__` and `loads` functions should have the same arguments so the underscore has been removed, apologies.

**Comment by yorikvanhavre** [comment]
Thanks!

**Pull request merged**